tools/Performance/DynamoPerformanceTests/DynamoModelPerformanceTestBase.cs

```diff
@@ -77,12 +38,43 @@ protected override void GetLibrariesToPreload(List<string> libraries)
             libraries.Add("FFITarget.dll");
         }
 
+        /// <summary>
+        /// Test directory containing the graph suite which the current performance test will be running against
+        /// </summary>
+        public static string testDirectory;
+
         /// <summary>
         /// Automated creation of performance test cases, one for each
-        /// parameter source.
+        /// parameter source. Notice this attribute must be public with
+        /// [ParamsSource] tag which will only thrown runtime error otherwise.
         /// </summary>
         [ParamsSource(nameof(PerformanceTestSource))]
-        public string DynamoFilePath { get; set; }
+        public static string DynamoFilePath { get; set; }
+
+        /// <summary>
+        /// Populates the test cases based on DYN files in the performance tests folder.
+        /// Notice this function must be public as well because it is 
+        /// defined as the ParamsSource of DynamoFilePath property.
+        /// Console app will throw runtime error otherwise.
+        /// </summary>
+        /// <returns></returns>
```

Comment:
Needs description of what is returned here

Reply:
Will do